### ISSUE TITLE
why is lifeStage "pup" excluded from the interpretation?

### ISSUE THREAD

**Issue by ymgan** [gbif/portal-feedback#5231]
Hello,

Why is the lifeStage "pup" excluded from the interpretation? Example: https://www.gbif.org/occurrence/4527130855

It is important (in my humble opinion) to distinguish phocid pup from other life stage such as juvenile for the downstream users because of their risk of predation, mortality, behaviours are different - this could be important for conservation effort. If this is a matter of interpretation of [vocabulary concepts](https://registry.gbif.org/vocabulary/LifeStage/concepts), can you please consider adding "pup" into the vocabulary or not nullify the field please?

**Comment by tucotuco** [comment]
Given that both juvenile and subadult are already in there, and less defensible as distinct concepts, I highly support this addition.

**Comment by CecSve** [comment]
Thank you for reporting the issue and we can add it. I was planning to include the [BODC vocabulary](http://vocab.nerc.ac.uk/collection/S11/current/) where pup is included. It might make sense to include the following as well:

- Cub
- Foal
- Calf

**Comment by CecSve** [comment]
I jumped the gun on this one. It is not a simple solution, as the concepts does not directly translate to other languages. For example, `pup` and `cub` are both `cachorro` in Spanish.

**Comment by MattBlissett** [comment]
(A reminder that I'm not a zoologist, so my knowledge of lifeStage terms is only from general use.)

As the intention is to distinguish between different stages of juvenile, and that's currently defined as an organism that hasn't reached adulthood, I think these would be different sub-concepts of that. We have this already for birds:

```
Chick
├── Hatchling
├── Nestling
└── Fledgeling
```

We could try and make general words for mammals, and add alternative names for these
```
Juvenile
├── Baby (= pre-weaned pup, = pre-weaned kitten, ...)
└── Child or something (= post-weaned pup, = post-weaned kitten, ... )
```

That would map everything to the general term (baby/child), which is easy for searching but not so nice for publishing or when looking at individual organisms.

I'll bet there are different numbers of relevant categories for different animals too (humans: neonate, infant, child, adolescent).

Or we could add vocabulary for each group, though it makes translation difficult:

```
Pup
├── Pre-weaned pup
└── Post-weaned pup
Kitten
├── Pre-weaned kitten
└── Post-weaned kitten
```

**Comment by CecSve** [comment]
I don't think that adding English concepts (only) for juvenile or sub-adult animals would be the best way to go about this. 

Currently, `cub` is a hidden value of `juvenile` and I could easily add `pup`, `post-weaning pup`, `pre-weaning pup`, `foal`, `lamb`, `calf`, etc. as hidden values. Then the interpreted value would be `juvenile` and I believe a user could use lifeStage+verbatim lifeStage and the family, genus, or species to filter their data, no? I can see some of the other BODC concepts are hidden values in the vocabulary as well, e.g. `smolt`. 

Regarding what we already have for birds/chicks:
```
Chick
├── Hatchling
├── Nestling
└── Fledgeling
```
This will most likely change after consulting with the NAOC work group. `Hatchling` will be removed as a concept and a `Not adult` concept will be included (not the same as `sub-adult`). 

Now, I realise that the level of detail currently in the vocabulary varies between organism groups: https://registry.gbif.org/vocabulary/LifeStage/concepts and I am not sure how to best structure this. 

Another plan is to include concept labels where relevant, indicating whether the concept is relevant for ornithology etc.

**Comment by ymgan** [comment]
Thank you so much! I appreciate the attempt to incorporate this into the [secret sauce](https://www.tdwg.org/conferences/2024/sessions/#sym13-controlled-vocabularies-the-secret-sauce-that-unlocks-the-power-of-data-consistency-and-accessibility)! 

I guess, this could also be a good use case for dwciri mentioned here: https://github.com/gbif/ipt/issues/1947

If user can specify

[dwciri:lifeStage](https://dwc.tdwg.org/terms/#dwciri:lifeStage): http://vocab.nerc.ac.uk/collection/S11/current/S1157/

Then we know that the cachorro refers to "pup". 

Anyway, thank you so much for your courage to take on this complex task!

**Comment by CecSve** [comment]
Thank you for sharing the dwciri thread - I had not seen that. 

We do have the option to specify URIs for equivalent concepts (see for example https://registry.gbif.org/vocabulary/Sex/concept/Male), however, the alternativeLabel_es (or in any other language) has to be unique for the concept. So we would still have the issue with `pup` and `cub` = `cachorro` if they were introduced as separate concepts. 

I will move this issue to the vocabulary repo to keep it in mind for future developments.
